src/main/java/uk/gov/pay/ledger/filters/LoggingMDCRequestFilter.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.pay.ledger.filters;
+
+import org.slf4j.MDC;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerRequestFilter;
+import java.io.IOException;
+import java.util.Optional;
+
+import static uk.gov.pay.logging.LoggingKeys.GATEWAY_ACCOUNT_ID;
+import static uk.gov.pay.logging.LoggingKeys.LEDGER_EVENT_ID;
+
+public class LoggingMDCRequestFilter implements ContainerRequestFilter {
+
+    public static final String PARENT_EXTERNAL_ID = "parent_external_id";
+    public static final String TRANSACTION_EXTERNAL_ID = "transactionExternalId";
```

Comment:
this will need to be `transaction_external_id` to follow the GDS json logging convention of snake case for keys.

Reply:
Done